packages/scoutgame/src/protocol/getNftsByOwner.ts

```diff
@@ -0,0 +1,100 @@
+import { getLastBlockOfWeek } from '@packages/blockchain/getLastBlockOfWeek';
```

Comment:
we are not using this method anywhere except my script but it might be useful in the future